core/src/zeit/connector/mock.py

```diff
@@ -53,6 +53,7 @@ def _reset(self):
         self._paths = {}
         self._deleted = set()
         self._properties = {}
+        self.search_result_default = self.search_result_default
```

Comment:
Das erreicht nicht die gewünschte Test-Isolierung. (Außerdem wärs für Clients eh netter `connector.search_result = [...]` zu schreiben, anstatt über `default` zu reden).

```suggestion
        self.search_result = self.search_result_default
```